### DISCUSSION TITLE
Is there any .ipynb tutorial is there to fine-tune Whisper for one language without making it lose its any of previous pretrained data?

### DISCUSSION THREAD

**Discussion by Singhamarjeet8130** [openai/whisper#2471] [Q&A]
I have successfully trained whisper, only to find to out that the all the previous pretrained data has been lost. 
So is there any .ipynb tutorial is available to fine-tune whisper for any supportive language without making it lose its any of the previous pretrained data?

**Comment by phineas-pta** [comment]
what previous pretrained data ?

**Reply by Singhamarjeet8130** [reply]
I tried to fine-tune whisper for Hindi language. So it should transcribe it perfectly but after fine-tuning. The fine-tuned model was only able to transcribe in hindi only even after putting audios of other languages. As whisper was already pre trained in multiple languages but after my fine-tuning. It's previous data was disappeared from it. I want to know if there is any way to fine-tune it for just one language without making it forget rest of languages that it remembers.

**Reply by Singhamarjeet8130** [reply]
Or is there any .ipynb tutorial available which serves my need the way I want to fine-tune whisper?

**Reply by phineas-pta** [reply]
after fine tuning, whisper ability to detect language drop catastrophically, do u specify lang when transcribing

**Reply by Singhamarjeet8130** [reply]
Hindi

**Reply by Singhamarjeet8130** [reply]
i specified the above language when transcribing

**Reply by phineas-pta** [reply]
so it works ?

**Reply by Singhamarjeet8130** [reply]
The training completed successfully but as you said that after fine tuning, whisper ability to detect language drop catastrophically and exactly this happened. So i have been searching for way to fine-tune whisper model for improving one language without making it lose its existing data of all other languages.
